Demo/src/main/java/com/paypal/android/ui/card/CardFragment.kt

```diff
@@ -169,11 +125,54 @@ class CardFragment : Fragment() {
     }
 
     private suspend fun createOrder() {
+
+        val accessToken = payPalDemoApi.fetchAccessToken().value
+        val configuration = CoreConfig()
```

Comment:
We could make `accessToken` a `CoreConfig` parameter to make authentication available to every SDK module via the core configuration.

Reply:
the only problem I see is that right now is only needed for `Card`. If we leave it on `CoreConfig`,the merchant might get confused and try to always put a token even though its not necessary

Reply:
The token will most likely be necessary for `v1`. A design goal would be to use the `accessToken` as the primary authentication mechanism, but it may not be that simple it sounds like GraphQL is a bit different than some of the PayPal API.